web/src/app.html

```diff
@@ -14,35 +14,96 @@
     <link rel="icon" type="image/png" sizes="144x144" href="/favicon-144.png" />
     <link rel="apple-touch-icon" sizes="180x180" href="/apple-icon-180.png" />
     %sveltekit.head%
+    <style>
+      /* prevent FOUC */
+      html {
+        height: 100%;
+        width: 100%;
+      }
+      body,
+      html {
+        margin: 0;
+        padding: 0;
+      }
+      @keyframes delayedVisibility {
+        to {
+          visibility: visible;
+        }
+      }
+      @keyframes stencil-pulse {
+        0% {
+          transform: scale(0.93);
+          filter: drop-shadow(0 0 0 rgba(0, 0, 0, 0.7));
+        }
+
+        70% {
+          transform: scale(1);
+          filter: drop-shadow(0 0 8px rgba(0, 0, 0, 0));
+        }
+
+        100% {
+          transform: scale(0.93);
+          filter: drop-shadow(0 0 0 rgba(0, 0, 0, 0));
+        }
+      }
+      @keyframes loadspin {
+        100% {
+          transform: rotate(360deg);
+        }
+      }
+      #stencil svg {
+        height: 35%;
+        animation: stencil-pulse 1s linear infinite;
+      }
+      #stencil {
+        --stencil-width: 25vw;
+        display: flex;
+        width: var(--stencil-width);
+        margin-left: auto;
+        margin-right: auto;
+        margin-top: calc(50vh - var(--stencil-width) / 2);
+        margin-bottom: 100vh;
+        place-items: center;
+        justify-content: center;
+        overflow: hidden;
+        visibility: hidden;
+        animation:
+          0s linear 0.3s forwards delayedVisibility,
+          loadspin 2s linear infinite;
+      }
+      .bg-immich-bg {
+        background-color: white;
+      }
+      .dark .dark\:bg-immich-dark-bg {
+        background-color: black;
+      }
+    </style>
     <script>
       /**
        * Prevent FOUC on page load.
        */
       const colorThemeKeyName = 'color-theme';
-      let theme;
-      const fallbackTheme = { value: 'dark', system: false };
-      let item = localStorage.getItem(colorThemeKeyName);
-      if (item === 'dark' || item === 'light') {
-        fallbackTheme.value = item;
-        item = JSON.stringify(fallbackTheme);
-        localStorage.setItem(colorThemeKeyName, item);
-      }
-
-      theme = JSON.parse(localStorage.getItem(colorThemeKeyName));
-
-      if (theme) {
-        if (theme.system) {
-          if (window.matchMedia && window.matchMedia('(prefers-color-scheme: dark)').matches) {
-            theme.value = 'dark';
-          } else {
-            theme.value = 'light';
-          }
-        }
+
+      let theme = localStorage.getItem(colorThemeKeyName);
+      if (!theme) {
+        theme = { value: 'light', system: true };
+      } else if (theme === 'dark' || theme === 'light') {
+        theme = { value: item, system: false };
```

Comment:
this line causes a JavaScript error `item is undefined` and one is unable to login. I suppose this was supposed to be `value: theme`?